src/main/java/com/example/draw/GraphGenerator.java

```diff
@@ -5,20 +5,27 @@
 import com.example.model.MyGraph;
 import org.springframework.stereotype.Component;
 
+import java.util.Map;
+
 @Component
 public class GraphGenerator {
 
-    public void generateGraph(GraphController graphController, DefinedGraph definedGraph, int numberOfVertices){
+    public void generateGraph(GraphController graphController, DefinedGraph definedGraph, Map<String, Integer> settings){
```

Comment:
I think that better option would be to return graph in this method and setModelGraph outside of this function. It would result in easier testing of this method (we wouldn't have to pass GraphController mock)

Reply:
Done.